### DISCUSSION TITLE
Last FFMPEG Commands

### DISCUSSION THREAD

**Discussion by mikenewsletters** [mifi/lossless-cut#606] [Q&A] [answered]
The help file says "The last executed ffmpeg commands will show up here after you run operations. You can copy them to clipboard and modify them to your needs before running on your command line.".

However, I don't see the commands appearing (Experimental is set).

**Comment by Shakil-Shahadat** [comment, marked as answer]
Check out developer tool from 'Tools > Toggle Developer Tools'.

**Reply by mikenewsletters** [reply]
That was quick!

Found it under "Console". 

Many thanks for the pointer.

**Comment by mifi** [comment]
That's odd. they should also appear under Help. Did you scroll all the way down?

**Reply by mikenewsletters** [reply]
Yep ... [Help] > [Help and shortcuts] shows "Last FFMPEG Commands, The last executed ffmpeg ... your command line". No command shown.

I'm using v3.23.8 (snaps) on Ubuntu.

**Reply by mifi** [reply]
Which operation did you run before that?

**Reply by mikenewsletters** [reply]
[Tools] > [Batch convert to supported format] > Selected file > [Selected "Fastest : Low playback speed"] > [Ok]

The file converts ok.

**Reply by mifi** [reply]
Ahh! The code currently only logs cutting related commands.
